src/fabric_cicd/constants.py

```diff
@@ -57,6 +57,7 @@
     "Lakehouse": "body/properties/sqlEndpointProperties/connectionString",
     "Warehouse": "body/properties/connectionString",
     "Eventhouse": "body/properties/queryServiceUri",
+    "SQLDatabase": "body/properties/connectionString",
```

Comment:
This will return the following: `"Data Source=fvzeldvqgvce3b2hxvbg4hnjqu-nowrtjcsie2e3ny6v4ojbd3esa.database.fabric.microsoft.com,1433;Initial Catalog=SQLDatabase1-45c6e7cf-89b1-4a69-b4e1-5495271c7e45;MultipleActiveResultSets=False;Connect Timeout=30;Encrypt=True;TrustServerCertificate=False"`

Is sqlendpoint misleading?